### DISCUSSION TITLE
posts/Go/grpc-gateway/

### DISCUSSION THREAD

**Discussion by Q1mi** [Q1mi/BlogComments#306] [Announcements]
https://www.liwenzhou.com/posts/Go/grpc-gateway/? 

 李文周的Blog提供免费的Go语言教程，本文是gRPC-Gateway的中文使用指南，演示了如何安装 gRPC-Gateway插件，通过为我们的protobuf文件添加注释从而为我们的RPC服务自动生成RESTful API代码。还详细介绍了如何在protobuf中导入google api的annotations.proto文件。

**Comment by bourne-3** [comment]
根据上述的方式搭建了gRPC-Gateway工程；希望能够统一返回码，例如返回仅有code、msg以及data三段式
请问得如何做呢
